examples/mongodbatlas_cloud_backup_schedule/main.tf

```diff
@@ -37,31 +34,45 @@ resource "mongodbatlas_advanced_cluster" "automated_backup_test_cluster" {
   }
 
   backup_enabled = true # enable cloud backup snapshots
+  pit_enabled    = true
 }
 
-resource "mongodbatlas_cloud_backup_schedule" "test" {
-  for_each     = local.atlas_clusters
-  project_id   = mongodbatlas_project.atlas-project.id
-  cluster_name = mongodbatlas_advanced_cluster.automated_backup_test_cluster[each.key].name
 
+resource "mongodbatlas_cloud_backup_schedule" "test" {
+  for_each                 = local.atlas_clusters
+  project_id               = mongodbatlas_project.atlas-project.id
+  cluster_name             = mongodbatlas_advanced_cluster.automated_backup_test_cluster[each.key].name
   reference_hour_of_day    = 3
   reference_minute_of_hour = 45
   restore_window_days      = 4
 
+  copy_settings {
+    cloud_provider = "AWS"
+    frequencies = ["HOURLY",
+      "DAILY",
+      "WEEKLY",
+      "MONTHLY",
+      "YEARLY",
+    "ON_DEMAND"]
+    region_name        = "US_WEST_1"
+    zone_id            = mongodbatlas_advanced_cluster.automated_backup_test_cluster[each.key].replication_specs[0].zone_id[0]
+    should_copy_oplogs = true
+  }
+
   policy_item_hourly {
     frequency_interval = 1 #accepted values = 1, 2, 4, 6, 8, 12 -> every n hours
     retention_unit     = "days"
-    retention_value    = 1
+    retention_value    = 4
```

Comment:
just curious what was the reason of changing these values? @AgustinBettati @maastha

Reply:
I see its came from this PR (https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2470/files#diff-5e7efdf36fb61aede763657eb676b950263932758699400e19562df38ffbe9e6R65), @maastha do you recall any additional context?

Reply:
These values were updated because the API was throwing an error with the previous values (i.e. that example was not working).
I don't remember the exact error code, but it required that the `retention_value` in the policy items should be greater than or equal to the `restore_window_days`  which is set to `restore_window_days = 4` in this example.


Please lmk if you have more questions, I can try to look up the exact error code.